### ISSUE TITLE
Provide a true new origin

### ISSUE THREAD

**Issue by hzeller** [hzeller/kicad-source-mirror#10]
The 'space' bar is only providing an origin for measuring, but it would be also good to place a true real origin on the board.
Entering coordinates for components or graphic lines such as for borders are using the absolute global coordinate system which might be pretty far off depending on where the board origins have been drawn.

Typically, boards are measured from one corner or one drill hole somewhere, so it would actually be good to actually re-center the origin at that point.
(there is already a way to do that, but AFAIK that only affects the Gerber origin).

**Comment by thatch** [comment]
Yes.  (There's Gerber/Drill origin, Grid Origin, and Space origin.  The first two, and your new origin, would typically be the same?)
